### PR TITLE
Generate changelog for 3.0.2 release

### PR DESCRIPTION
**Pull request by sergos**
Includes cleanup for changelogs

### CODE REVIEW

File: changelogs/unreleased/gh-9399-raft-leader-steps-off-on-ER_WAL_IO.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## feature/raft
+## bugfix/raft
```
Comment (Totktonada) [inline changelogs/unreleased/gh-9399-raft-leader-steps-off-on-ER_WAL_IO.md]: @sergepetrenko Please, take a look.

Comment (sergepetrenko) [reply changelogs/unreleased/gh-9399-raft-leader-steps-off-on-ER_WAL_IO.md]: I'm ok. Thanks for noticing!

**Review by Totktonada — APPROVED** [review]
I see no flaws.

**Review by sergepetrenko — APPROVED** [review]

**Review by ylobankov — APPROVED** [review]

**Review by lenkis — APPROVED** [review]
Looks good to me

**Pull request merged**